src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
+
+const isObject = (value) => typeof value === 'object' && value !== null;
+
+const getAverageAge = people => isObject(people[0])
+
+  ? people.reduce((acc, { born, died }) =>
+    acc + (died - born), 0) / people.length || 0
+
+  : people.reduce((total, value) => total + value, 0);
```

Comment:
Why are you checking if the first element is of type "object"?
1. The object and the array will be of this type: `typeof {} === 'object'` / `typeof [] === 'object'` -  therefore, this check makes no sense.
2. In the second `reduce`, you do not return the average value, but only calculate the sum, so the name `getAverageAge` does not fit here and it can be transferred to a separate function.